tokio/src/util/metric_atomics.rs

```diff
@@ -45,3 +45,32 @@ impl MetricAtomicU64 {
         pub(crate) fn new(_value: u64) -> Self { Self { } }
     }
 }
+
+cfg_rt! {
+    /// `AtomicUsize` for use in metrics.
+    ///
+    /// This exposes simplified APIs for use in metrics & uses `std::sync` instead of Loom to avoid polluting loom logs with metric information.
+    #[derive(Debug, Default)]
+    pub(crate) struct MetricAtomicUsize {
+        value: std::sync::atomic::AtomicUsize,
+    }
```

Comment:
Does this need to be gated on `cfg_rt!`? Is the file not already gated on that?

Reply:
these are only used from the `rt` module (by coincidence, the other metrics are all u64) so I gated on rt to avoid dead code warnings.

Reply:
Can you add an `#[cfg_attr(not(feature = "rt"), allow(dead_code))]` on the module instead? The `cfg_rt!` blocks break rustfmt, so I try to avoid wrapping large blocks in them.

Reply:
Or maybe it'd be better to split the file into two and wrap the `mod` statement in `cfg_rt!`.